cmd/facette/template/browse/graph.html

```diff
@@ -43,6 +43,23 @@
 								</div>
 							</div>
 						</li>
+						<li>
+							<a class="icon icon-size" href="#set-global-size" title="Set graph dimensions"></a>
+							<div id="widthmenu" class="menu">
+								<div class="menucntr">
+									<div class="menumesg">Set Graph Width: <em id="widthlabel">1x</em></div>
+									<div class="menuitem">
+										<input type="range" name="width" class="size-slider" step="1"
+										    min="1" max="10" value="1" oninput="$('#widthlabel').html(this.value + 'x');"/>
```

Comment:
Please avoid using Javascript directly in the HTML source. Try to bind the events directly from the js files.